File: Orbit.Domain/DTOs/Assets/CreateAssetRequest.cs
namespace Orbit.Domain.DTOs.Assets
{
    public class CreateAssetRequest
    {
        public required string AssetName { get; set; }
        public string? Brand { get; set; }
        public string? Model { get; set; }
        public string? SerialNumber { get; set; }
        public required DateTime PurchaseDate { get; set; }
        public decimal? PurchasePrice { get; set; }
        public string? Location { get; set; }
        public DateTime? WarrantyExpirationDate { get; set; }
        public string? Notes { get; set; }
        public required string Status { get; set; }
        public required int CategoryId { get; set; }
    }
}

File: orbit.web/interfaces/api/assets/CreateAssetRequest.ts
export interface CreateAssetRequest {
  assetName: string
  brand?: string | null
  model?: string | null
  serialNumber?: string | null
  purchaseDate: string
  purchasePrice?: number | null
  location?: string | null
  warrantyExpirationDate?: string | null
  notes?: string | null
  status: string
  categoryId: number
}
